### PR TITLE
[Cult Balance - March 2019] Pulling stuff through cult doors

### PR DESCRIPTION
**Pull request by DeityLink**
* You can now pull stuff through cult doors (they previously closed immediately, preventing you from dragging stuff through, which is pretty irritating)
* Updated the sacrifice objective's explanation text to remind cultists that they can soulstone the people they sacrifice by using an empty soul blade.
* Bloodstones spawning will no longer harm soul blades (especially those that got used during a sacrifice)

### CODE REVIEW

**Comment by DeityLink** [comment]
@MadmanMartian I targetted the wrong branch again.

File: code/datums/gamemode/factions/bloodcult/bloodcult_effects.dm [hunk 1]
```diff
@@ -420,3 +420,108 @@ var/list/bloodstone_backup = 0
...
+	mouse_opacity = 0
+	var/list/viewers = list()
+	var/image/indicator = null
+	var/current_dots = 6
+	var/mob/victim = null
+
+/obj/effect/stun_indicator/New()
+	..()
+	if (!ismob(loc))
+		qdel()
```
Comment (DamianX) [inline code/datums/gamemode/factions/bloodcult/bloodcult_effects.dm]: you mean `qdel(src)`

File: code/datums/gamemode/factions/bloodcult/bloodcult_effects.dm [hunk 2]
```diff
@@ -420,3 +420,108 @@ var/list/bloodstone_backup = 0
...
+		return
+
+	victim = loc
+	current_dots = Clamp(round(victim.knockdown/2.5),0,5)
+
+	if (!current_dots)
+		qdel()
```
Comment (DamianX) [inline code/datums/gamemode/factions/bloodcult/bloodcult_effects.dm]: you mean `qdel(src)`

**Comment by Kevak** [comment]
I vaguely recall about a week ago, planting an empty soulblade on a sac target when they were on the alter, and when they were sacced, it left behind an empty soulshard and no blade beneath the bloodstone. 

You certain that procset is working? I know the sac target in question was not cultbanned, for I've seen em as a cultie on no fewer than four occasions since then.

**Comment by DeityLink** [comment]
Oh shit, I think the bloodstone spawning might have pulverized the blade

**Comment by Kevak** [comment]
That's what I assumed yah. Cause I know a gem being qdeled makes a shard.

**Pull request merged**